test/e2e/suite_test.go

```diff
@@ -377,8 +377,8 @@ func applyPollingDefaults() {
 	polling.stargateReady.timeout = 5 * time.Minute
 	polling.stargateReady.interval = 5 * time.Second
 
-	polling.reaperReady.timeout = 5 * time.Minute
-	polling.reaperReady.interval = 5 * time.Second
+	polling.reaperReady.timeout = 7 * time.Minute
+	polling.reaperReady.interval = 10 * time.Second
```

Comment:
Is it an attempt to fix the reaper issue with secrets not being managed by K8ssandra?
Doesn't seem to work if it's the case looking at the test results.

Reply:
I increased the timeout because in one e2e failure I was looking at, Reaper was initializing but it actually appeared to take > 5 minutes.